pro/src/screens/OfferIndividual/StocksEvent/StockFormList/StockFormList.tsx

```diff
@@ -60,8 +62,8 @@ const StockFormList = ({ offer, onDeleteStock }: IStockFormListProps) => {
 
           <div className={styles['form-list']}>
             {values.stocks.map((stockValues: IStockEventFormValues, index) => (
-              <StockFormRow
-                key={index}
+              <StockEventFormRow
+                stockIndex={index}
```

Comment:
il faut toujours un key pour react dans un map quand on render des composants,
si possible une qui vient des données et pas du map (ça lui permet de gérer bien les rerender il me smble)

https://reactjs.org/docs/lists-and-keys.html#keys